src/coreclr/jit/hwintrinsiclistarm64.h

```diff
@@ -155,7 +155,7 @@ HARDWARE_INTRINSIC(Vector128,     EqualsAny,
 HARDWARE_INTRINSIC(Vector128,     ExtractMostSignificantBits,                                       16,      1,     false,  {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_Helper,                HW_Flag_InvalidNodeId|HW_Flag_BaseTypeFromFirstArg)
 HARDWARE_INTRINSIC(Vector128,     Floor,                                                            16,      1,     false,  {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_Helper,                HW_Flag_InvalidNodeId)
 HARDWARE_INTRINSIC(Vector128,     FusedMultiplyAdd,                                                 16,      3,     false,  {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_Helper,                HW_Flag_InvalidNodeId)
-HARDWARE_INTRINSIC(Vector128,     GetElement,                                                       16,      2,      true,  {INS_smov,           INS_umov,           INS_smov,           INS_umov,           INS_smov,           INS_umov,           INS_umov,           INS_umov,           INS_dup,            INS_dup},         HW_Category_Helper,                HW_Flag_SpecialImport|HW_Flag_SpecialCodeGen|HW_Flag_BaseTypeFromFirstArg)
+HARDWARE_INTRINSIC(Vector128,     GetElement,                                                       16,      2,      true,  {INS_smov,           INS_umov,           INS_smov,           INS_umov,           INS_smov,           INS_umov,           INS_umov,           INS_umov,           INS_dup,            INS_dup},         HW_Category_Helper,                HW_Flag_SpecialImport|HW_Flag_SpecialCodeGen|HW_Flag_BaseTypeFromFirstArg|HW_Flag_SupportsContainment)
```

Comment:
one thing I noticed that GetElement is not marked as `HW_Flag_SupportsContainment` for x64 but it still works there 🤔

Reply:
They’re functionally inverses of each other.

On Arm64 containment is a rare consideration and so is opt-in, while on x64 it’s a common thing for almost every operation and is instead opt-out